pkg/tsdb/azuremonitor/macros_test.go

```diff
@@ -0,0 +1,155 @@
+package azuremonitor
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/tsdb"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAzureLogAnalyticsMacros(t *testing.T) {
+	fromStart := time.Date(2018, 3, 15, 13, 0, 0, 0, time.UTC).In(time.Local)
+	timeRange := &tsdb.TimeRange{
+		From: fmt.Sprintf("%v", fromStart.Unix()*1000),
+		To:   fmt.Sprintf("%v", fromStart.Add(34*time.Minute).Unix()*1000),
+	}
+
+	tests := []struct {
+		name      string
+		query     *tsdb.Query
+		timeRange *tsdb.TimeRange
+		kql       string
+		expected  string
+		Err       require.ErrorAssertionFunc
+	}{
+		{
+			name:      "invalid macro should throw error",
+			timeRange: timeRange,
```

Comment:
This value is the same for each test case. Perhaps you can remove it from the test struct and use this value in the test loop?

Reply:
Good catch. I had another test case originally that used it but I removed it and missed cleaning up.